### PR TITLE
[feat]curve: run deamon after start

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Cyber-SiKu** [comment]
cicheck

**Review by Wine93 — APPROVED** [review]

File: nebd/src/part2/file_entity.cpp [hunk 1]
```diff
@@ -361,6 +363,11 @@ int NebdFileEntity::UpdateFileStatus(NebdFileInstancePtr fileInstance) {
     fileInstance_ = fileInstance;
     status_ = NebdFileStatus::OPENED;
     timeStamp_ = TimeUtility::GetTimeofDayMs();
+    OpenFlags flags;
+    if (fileInstance->xattr.count(kOpenFlagsAttrKey) &&
+        flags.ParseFromString(fileInstance->xattr.at(kOpenFlagsAttrKey))) {
+        openFlags_.reset(new OpenFlags{flags});
+    }
```
Comment (wu-hanqing) [inline nebd/src/part2/file_entity.cpp:370]: it seems this code duplicates with L121-L125

Comment (Cyber-SiKu) [reply nebd/src/part2/file_entity.cpp]: > it seems this code duplicates with L121-L125

fix

**Comment by Cyber-SiKu** [comment]
cicheck

**Pull request merged**